src/main/java/org/sqlite/Function.java

```diff
@@ -127,8 +123,8 @@ public static final void create(Connection conn, String name, Function f, int nA
      * @param nArgs The number of args for the function.
      * @throws SQLException
      */
-    public static final void destroy(Connection conn, String name, int nArgs) throws SQLException {
```

Comment:
why remove the `final` here?

Reply:
Hey. Sorry for the long time to respond - it's been busy. I removed it because my IDE (IntelliJ) suggested it is useless for a method to be both static and final.

I now see that this isn't entirely true. If you have a class with a static final method, a subclass that implements its own static final method of the same name won't compile. It's a silly use case though, because there is no polymorphism for static methods, so preventing a subclass from overloading create with these exact parameters doesn't seem to serve much purpose.

Nonetheless, I see that there are this syntax does change the code meaning, and as I'm not the author of this codebase it's not my place to make code style decisions and this definitely shouldn't be packaged into a bugfix MR. I will remove this from the PR shortly.

With that said, I am curious to know the rational for adding this keyword here. Is it just good style to denote that this method shouldn't be changed? Or maybe some IDEs offer to override this static function (in error) and final helps them know not to?